shell/platform/android/test/io/flutter/embedding/android/FlutterViewTest.java

```diff
@@ -375,6 +378,59 @@ public void systemInsetHandlesFullscreenNavbarLeft() {
     assertEquals(100, viewportMetricsCaptor.getValue().paddingRight);
   }
 
+  @Test
+  @TargetApi(30)
+  @Config(sdk = 30)
```

Comment:
Does this actually work? I'm somewhat surprised it doesn't complain about wanting a new version of the robolectric dependencies

Reply:
Seems like it does, test passes and it can only do so if running on API 30.